.travis.yml

```diff
@@ -1,15 +1,62 @@
-language: python
-install: pip install -r requirements_setup.txt -r requirements_test.txt
+---
+language: bash
+sudo: required
+dist: xenial
+
+before_install:
+- sudo apt-get -qq update
+- sudo apt-get install -y "rpm" "debhelper" python-all fakeroot python-setuptools
+  jq
```

Comment:
This looks odd on a line on its own. Do you need to make a YAML multi-line string with '>'?

Reply:
Yes, I do. Done, testing.